src/vivarium_nih_us_cvd/components/treatment.py

```diff
@@ -837,6 +841,48 @@ def enroll_in_polypill(
 
         return pop_visitors
 
+    def apply_lifestyle_ramp(self, pop_visitors: pd.DataFrame) -> pd.DataFrame:
+        # Determine testing eligibility
+        not_already_enrolled = pop_visitors[data_values.COLUMNS.LIFESTYLE].isna()
+        bmi = self.bmi(pop_visitors.index)
+        age = pop_visitors["age"]
+
+        fpg_not_tested_recently = (
+            # never been tested for FPG
+            pop_visitors[data_values.COLUMNS.LAST_FPG_TEST_DATE].isna()
+        ) | (
+            # last FPG test more than 3 years ago
+            pop_visitors[data_values.COLUMNS.LAST_FPG_TEST_DATE]
+            < self.clock()
+            - pd.Timedelta(days=365.25 * data_values.FPG_TESTING.NUM_YEARS_BEFORE_SIM_START)
```

Comment:
Nit: I think you should rename `NUM_YEARS_BEFORE_SIM_START` to something like `MIN_YEARS_BETWEEN_TESTS`